src/components/Calculator.js

```diff
@@ -1,26 +1,39 @@
-      <button type="button" className="multi orange">x</button>
-      <button type="button" className="four">4</button>
-      <button type="button" className="five">5</button>
-      <button type="button" className="six">6</button>
-      <button type="button" className="substract orange">-</button>
-      <button type="button" className="one">1</button>
-      <button type="button" className="two">2</button>
-      <button type="button" className="three">3</button>
-      <button type="button" className="plus orange">+</button>
-      <button type="button" className="zeroo">0</button>
-      <button type="button" className="dott">.</button>
-      <button type="button" className="equal orange">=</button>
+      <div className="result"><p>{result.next || result.operation || result.total || '0'}</p></div>
+      <button type="button" className="AC" onClick={(e) => buttonHandler(e.target.textContent)}>AC</button>
+      <button type="button" className="plusnsub" onClick={(e) => buttonHandler(e.target.textContent)}>+/-</button>
+      <button type="button" className="percentage" onClick={(e) => buttonHandler(e.target.textContent)}>%</button>
+      <button type="button" className="division orange" onClick={(e) => buttonHandler(e.target.textContent)}>÷</button>
+      <button type="button" className="seven" onClick={(e) => buttonHandler(e.target.textContent)}>7</button>
+      <button type="button" className="eight" onClick={(e) => buttonHandler(e.target.textContent)}>8</button>
+      <button type="button" className="nine" onClick={(e) => buttonHandler(e.target.textContent)}>9</button>
+      <button type="button" className="multi orange" onClick={(e) => buttonHandler(e.target.textContent)}>x</button>
+      <button type="button" className="four" onClick={(e) => buttonHandler(e.target.textContent)}>4</button>
+      <button type="button" className="five" onClick={(e) => buttonHandler(e.target.textContent)}>5</button>
+      <button type="button" className="six" onClick={(e) => buttonHandler(e.target.textContent)}>6</button>
+      <button type="button" className="substract orange" onClick={(e) => buttonHandler(e.target.textContent)}>-</button>
+      <button type="button" className="one" onClick={(e) => buttonHandler(e.target.textContent)}>1</button>
+      <button type="button" className="two" onClick={(e) => buttonHandler(e.target.textContent)}>2</button>
+      <button type="button" className="three" onClick={(e) => buttonHandler(e.target.textContent)}>3</button>
+      <button type="button" className="plus orange" onClick={(e) => buttonHandler(e.target.textContent)}>+</button>
+      <button type="button" className="zeroo" onClick={(e) => buttonHandler(e.target.textContent)}>0</button>
+      <button type="button" className="dott" onClick={(e) => buttonHandler(e.target.textContent)}>.</button>
+      <button type="button" className="equal orange" onClick={(e) => buttonHandler(e.target.textContent)}>=</button>
```

Comment:
To follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). I would suggest that we reduce the number of duplicated codes to follow the DRY principle (Do not Repeat Yourself) 🤓